.github/scripts/cleanup_pr_body.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# ensure 1 argument is passed
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <pr_number>"
+    exit 1
+fi
+
+PR_NUMBER=$1
+OLD=/tmp/orig_pr_body.txt
+NEW=/tmp/new_pr_body.txt
+
+gh pr view --json body --template "{{.body}}" "${PR_NUMBER}" > "${OLD}"
+cp "${OLD}" "${NEW}"
+
+# Remove all lines after and including "**BEFORE SUBMITTING, PLEASE READ THE CHECKLIST BELOW AND FILL IN THE DESCRIPTION ABOVE**"
+sed -i '/\*\*BEFORE SUBMITTING, PLEASE READ THE CHECKLIST BELOW AND FILL IN THE DESCRIPTION ABOVE\*\*/,$d' "${NEW}"
```

Comment:
I'm also wondering if it's be easier and more future-proof if we just match for the contents of `.github/PULL_REQUEST_TEMPLATE.md` and remove that.

Reply:
I thought about that. The challenge is that sometimes people only change the "FILL IN THE PR DESCRIPTION HERE" line, but leave the rest of the template in place. That's why I split it into a few chunks.

Reply:
I tested this across several PRs that had different variations to get a result across common patterns I've seen.